### PR TITLE
Fix typo in readme

### PR DESCRIPTION
**Pull request by frinkleko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ N_DEVICES=8
 torchrun --nproc_per_node=${N_DEVICES} train_ir.py config_infonce.yaml
 ```
 
-Data file is a datasets.Dataset, each sample is an object like this:
```
Comment (zhichao-aws) [inline README.md]: the datasets here is package name https://huggingface.co/docs/datasets/index
I think we don't need to do change here

File: README.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ The data file has the same format as training with infoNCE.
 ## Run with knowledge distillation (scores pre-computed)
 For expensive teacher models like LLM or cross-encoders, we can calculate the scores in advance and store the scores. To run with pre-computed KD scores, use_in_batch_negatives should be set to false.
 
-Data file is a datasets.Dataset, each sample is an object like this:
+Data file is a dataset. Dataset, each sample is an object like this:
```
Comment (zhichao-aws) [inline README.md]: same as above

**Review by zhichao-aws — COMMENTED** [review]
Thanks for the fix :)

File: README.md (hunk 1 above)
Comment (frinkleko) [reply README.md]: Sure, I had some misunderstanding.

**Review by zhichao-aws — APPROVED** [review]

**Pull request merged**